From 9ba813cd6963c8a987566499032ab633a955922c Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Tue, 10 Dec 2013 23:32:30 +0100
Subject: [PATCH 279/279] CLEANUP: check: server port is unsigned

Baptiste Assmann reported some confusing printf() output of the server
port since it's declared signed. Better turn it to unsigned.

There's no need to backport this, it's only used in 16-bit places.
---
 include/types/server.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/types/server.h b/include/types/server.h
index 82eb55f..05d4783 100644
--- a/include/types/server.h
+++ b/include/types/server.h
@@ -113,7 +113,7 @@ struct tree_occ {
 struct check {
 	struct connection *conn;		/* connection state for health checks */
 
-	short port;				/* the port to use for the health checks */
+	unsigned short port;			/* the port to use for the health checks */
 	struct buffer *bi, *bo;			/* input and output buffers to send/recv check */
 	struct task *task;			/* the task associated to the health check processing, NULL if disabled */
 	struct timeval start;			/* last health check start time */
-- 
1.8.1.5

